vendor/github.com/pingcap/kvproto/pkg/metapb/metapb.pb.go

```diff
@@ -181,10 +181,11 @@ func (m *RegionEpoch) GetVersion() uint64 {
 type Region struct {
 	Id uint64 `protobuf:"varint,1,opt,name=id,proto3" json:"id,omitempty"`
 	// Region key range [start_key, end_key).
-	StartKey    []byte       `protobuf:"bytes,2,opt,name=start_key,json=startKey,proto3" json:"start_key,omitempty"`
-	EndKey      []byte       `protobuf:"bytes,3,opt,name=end_key,json=endKey,proto3" json:"end_key,omitempty"`
-	RegionEpoch *RegionEpoch `protobuf:"bytes,4,opt,name=region_epoch,json=regionEpoch" json:"region_epoch,omitempty"`
-	Peers       []*Peer      `protobuf:"bytes,5,rep,name=peers" json:"peers,omitempty"`
+	StartKey           []byte       `protobuf:"bytes,2,opt,name=start_key,json=startKey,proto3" json:"start_key,omitempty"`
+	EndKey             []byte       `protobuf:"bytes,3,opt,name=end_key,json=endKey,proto3" json:"end_key,omitempty"`
+	RegionEpoch        *RegionEpoch `protobuf:"bytes,4,opt,name=region_epoch,json=regionEpoch" json:"region_epoch,omitempty"`
+	Peers              []*Peer      `protobuf:"bytes,5,rep,name=peers" json:"peers,omitempty"`
+	LastSplitTimestamp int64        `protobuf:"varint,6,opt,name=last_split_timestamp,json=lastSplitTimestamp,proto3" json:"last_split_timestamp,omitempty"`
```

Comment:
why did `gopkg.toml` not change?

Reply:
just copy. When related kvproto PR merged, I will update it.